agora-scripts/Main.hs

```diff
@@ -97,17 +97,19 @@ agoraScripts params =
 
     scripts = Bootstrap.agoraScripts plutarchConfig governor
 
+    plutarchConfig :: Config
+    plutarchConfig = Config {tracingMode = if params.tracing then DoTracing else NoTracing}
+
 {- | Params required for creating script export.
 
-     @since 0.2.0
+     @since 1.0.0
 -}
-data ScriptParams where
-  ScriptParams ::
-    { governorInitialSpend :: TxOutRef
-    , gtClassRef :: Tagged GTTag AssetClass
-    , maximumCosigners :: Integer
-    } ->
-    ScriptParams
+data ScriptParams = ScriptParams
+  { governorInitialSpend :: TxOutRef
+  , gtClassRef :: Tagged GTTag AssetClass
+  , maximumCosigners :: Integer
+  , tracing :: Bool
```

Comment:
This will eventually be handled in `plutarch-script-export`, right?

Reply:
I mean, it seems to be more reasonable to make it a command line option.

Reply:
(Or belong to a parent json object)

Reply:
It can't really be done in `plutarch-script-export` as it currently stands. Still, it should be possible to get both from within the offchain on demand, I'd think.

Reply:
Hmmm, I'll redo this a little.

Reply:
Due to the fact that offchain code can't use two versions of scripts simultaneously, having the ability to supply to versions just feels weird to me. Also, we will have to implement the same command line toggle in agora-pro.

Reply:
Yeah, you're right. I'll see if I can rework it a little bit. Might be a breaking change though.

Reply:
Hmmm. I can't figure out anything satisfactory yet. I think we're better off just going with this for now.

Reply:
Yeah, fair enough.